dags/dbt_daily.py

```diff
@@ -27,6 +28,8 @@
     "email_on_retry": True,
     "retries": 2,
     "retry_delay": timedelta(minutes=30),
+    "dbt_cloud_conn_id": "dbt_cloud",
+    "account_id": Variable.get("dbt_account_id")
```

Comment:
I think best practice is to retrieve variables at runtime so it doesn't do a database query on each dag parse https://www.astronomer.io/docs/learn/airflow-variables#using-environment-variables
```suggestion
    "account_id": "{{ var.value.dbt_account_id }}"
```